.github/workflows/pythonpackage.yml

```diff
@@ -43,8 +43,32 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ["3.6", "3.10"]
+        include:
```

Comment:
I would suggest dropping 3.6 support instead. It reached end of life status one year ago.
Keep using the matrix, drop 3.6, add 3.11.

Reply:
ok, done. unfortunately, python 3.11 does not seem to work with our test setup, and the error message is rather cryptic:

https://github.com/cantools/cantools/actions/runs/3674907063/jobs/6213690296

maybe you have an idea? (I stayed on 3.10 in the meantime.)